src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -30,7 +30,7 @@ public ApplicationExtract(ILogger<ApplicationExtract> log, ApplyDataContext appl
         }
 
 
-        [FunctionName("ApplicationExtract")]
+        //[FunctionName("ApplicationExtract")]
```

Comment:
remove comment?

Reply:
Good spot! Had commented it out for debugging. It's now reinstated.